models/believer/believer.sdf

```diff
@@ -0,0 +1,693 @@
+<sdf version='1.6'>
+  <model name='believer'>
+    <pose>0 0 0.246 0 0 0</pose>
+    <link name='base_link'>
+      <pose>0 0 0 0 -0 0</pose>
+      <inertial>
+        <pose>0.15 0 0.1 0 -0 0</pose>
+        <mass>2.05</mass>
+        <inertia>
+          <ixx>0.16632</ixx>
+          <ixy>0</ixy>
+          <ixz>0.0755</ixz>
+          <iyy>0.3899</iyy>
+          <iyz>0</iyz>
+          <izz>0.5243</izz>
+        </inertia>
+      </inertial>
+      <collision name='base_link_collision'>
+        <pose>0 0 -0.07 0 0 0</pose>
+        <geometry>
+          <box>
+            <size>1.088 2.591 0.103</size>
+          </box>
+        </geometry>
+        <surface>
+          <contact>
+            <ode>
+              <max_vel>10</max_vel>
```

Comment:
any reason for this?

Reply:
This allows the model to slide on the floor for takeoff